doc/user/content/cloud/settings.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Settings"
+description: "Change Materialize Cloud profile or password."
+menu:
+  main:
+    parent: "cloud"
+    weight: 4
+---
+
+{{< cloud-notice >}}
+
+The settings enable you to update your personal details and change your password, and enable [administrative users](../administer-workspace) to change workspace names and add or remove new users.
+
+To go to your Materialize Cloud settings, click on the user icon in the right of the top navbar.
+
+## Profile
+
+On the Profile tab, you change any of the following:
+
+* Email address
+* Name
+* Phone number
+* Address
+* Job title
+
+You can also delete a profile if you wish to delete your Materialize Cloud account. If you are the sole administrator of any workspaces, when you delete your profile, you will be asked whether you want to delete the workspace as well or give administrator privileges to another user.
```

Comment:
Heh, you actually can't do this right now! Maybe: "email us if you need to delete your account"?